chart/githubconnector/README.md

```diff
@@ -10,6 +23,68 @@ In order to install the chart inside of kyma you need to:
 * be connected to your kyma instance
 * have a properly configured chart
 
-## Details
+## Installing in Kyma using Helm
```

Comment:
This doc also needs some love - proper capitalization, using `code` formatting for all flags and paths, using note formatting that follows the Kyma standards. 
Additionally, make sure to write heading sentences in imperative, as in: `Register a service`.